specs/CustomStoragePartition.md

```diff
@@ -0,0 +1,208 @@
+Custom Storage Partition
+===
+
+# Background
+For certain WebView host apps, there is a desire to have different contexts for
+different WebViews using the same profile so that when the same websites run in
+these WebViews, they will have different DOM storages and cookie jars. An example
+is different accounts within an application.
+Previosly, the application can use different profiles for different contexts, but
```

Comment:
For two accounts within an application under the same profile, the only security threat is the storage/cookies?

Reply:
That's the only concern we perceive today and some first party app is asking for.

Reply:
Please ensure the reference documentation has any additional appropriate security related information.